### PR TITLE
First branch

### PR DESCRIPTION
**Pull request by PaulJame5**
Hey, I have the basics of everything set down. Never set a game over for when lives hit 0 or when you destroy all asteroids.

### CODE REVIEW

File: core/src/main/java/com/mystudio/gamename/Game.java [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+        }
+        ui.set(lives.getRemainingLives(),score.getScore());
+        asteroid.setActive(false);
+        activeAsteroids.remove(asteroid);
+    }
+
+    public void placeAsteroid(int from, int to, float posX, float posY) {
+        int numberOfAsteroidsSpawned = 0;
+        for (int i = from; i < to; i++) {
+            if (!asteroids[i].isActive()) {
```
Comment (tomcashman) [inline core/src/main/java/com/mystudio/gamename/Game.java]: This should change to fail fast.

File: core/src/main/java/com/mystudio/gamename/GameSprite.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            anim.addFrame(spriteSheet.getSprite(i), frameDuration);
+        }
+    }
+
+    public Animation getAnimation() {
+        return anim;
+    }
+
+    public int getFrameWidth()
+    {
```
Comment (tomcashman) [inline core/src/main/java/com/mystudio/gamename/GameSprite.java]: This class needs to be formatted

**Review by tomcashman — CHANGES_REQUESTED** [review]

**Comment by tomcashman** [comment]
Commits should be squashed when sending a pull request. See info [here](https://github.com/wprig/wprig/wiki/How-to-squash-commits).

**Comment by PaulJame5** [comment]
Made the changes, will close and open a new pull request

**Pull request closed without merging**